openpbr.bib

```diff
@@ -3,5 +3,7 @@ @techreport{OpenPBR
   title = {OpenPBR Surface},
   institution = {Academy Software Foundation (ASWF)},
   year = {2024},
+  month = {May},
+  version = {1.0},
```

Comment:
A note of caution to @jstone-lucasfilm for the release candidate, that we now have a third place where the version number appears.

Reply:
I'm not convinced a version number is needed in a citation. It can be assumed that the reference will want to refer to the latest version, which is available at the given link.

Reply:
Agreed that the version number is not usually included in citations, so this may not end up being an issue in practice.

Reply:
I'll remove it for now.